enterprise_access/apps/subsidy_request/psg_tasks.py

```diff
@@ -0,0 +1,108 @@
+"""
+Tasks for subsidy requests app.
+"""
+
+import logging
+from datetime import datetime
+
+from celery import shared_task
+from django.apps import apps
+from django.conf import settings
+from requests.exceptions import HTTPError
+
+from enterprise_access.apps.api_client.braze_client import BrazeApiClient
+from enterprise_access.apps.api_client.discovery_client import DiscoveryApiClient
+from enterprise_access.apps.api_client.lms_client import LmsApiClient
+from enterprise_access.apps.subsidy_request.constants import SubsidyRequestStates
+from enterprise_access.tasks import LoggedTaskWithRetry
+from enterprise_access.utils import get_subsidy_model
+
+logger = logging.getLogger(__name__)
+
+@shared_task(base=LoggedTaskWithRetry)
+def psg_send_admins_email_with_new_requests_task(enterprise_customer_uuid):
+    """
+    Task to send new-request emails to admins.
+
+    Args:
+        enterprise_customer_uuid (str): enterprise customer uuid identifier
+    Raises:
+        HTTPError if Braze client callfails with an HTTPError
+    """
+    lms_client = LmsApiClient()
+    enterprise_customer_data = lms_client.get_enterprise_customer_data(enterprise_customer_uuid)
+
+    config_model = apps.get_model('subsidy_request.SubsidyRequestCustomerConfiguration')
+    customer_config = config_model.objects.get(
+        enterprise_customer_uuid=enterprise_customer_uuid,
+    )
+
+    subsidy_model = get_subsidy_model(customer_config.subsidy_type)
+    subsidy_requests = subsidy_model.objects.filter(
+        enterprise_customer_uuid=enterprise_customer_uuid,
+        state=SubsidyRequestStates.REQUESTED,
+    )
+    # Filter when we last run this unless we never ran before
+    # "future" is greater than "past"
+    # so if created is greater than last remind date, it means
+    # it was created after cron was last run
+    if customer_config.last_remind_date is not None:
+        subsidy_requests = subsidy_requests.filter(created__gte=customer_config.last_remind_date)
+
+    subsidy_requests = subsidy_requests.order_by("-created")
+
+    if not subsidy_requests:
```

Comment:
I _think_ you want to use `exists` here. I'm not sure if this will evaluate correctly (i.e. is the QuerySet evaluated by such a statement?). (If I had more time, I'd just look this up myself.)